UE/Application/Application.hpp

```diff
@@ -24,7 +24,12 @@ class Application : public IEventsHandler
     // IUserEventsHandler interface
     void handleShowSmsList() override;
     void handleShowSms(IUeGui::IListViewMode::Selection) override;
-
+    void handleSendCallAccept(common::PhoneNumber to) override;
+    void handleSendCallDrop(common::PhoneNumber to) override;
+    void handleComposeSms() override;
+    void handleSendSms(const Sms& sms) override;
+    void handleStartDial() override;
+    void handleSendCallRequest(common::PhoneNumber from, common::PhoneNumber to) override;
```

Comment:
```from``` is **probably** not needed, BtsPort has phoneNumber member variable.

Reply:
Also, I don't know if `from `is necessary for the same reason, but in the specification _4.1.7.1 Sending Call Request - Use Case_ it is clearly written that sendCallRequest from UE to BTS contains `from` and `to `field. That's why I'll leave it that way for now.